docs/supported-tools.rst

```diff
@@ -0,0 +1,246 @@
+
+pip:
+  Latest version by default.
+
+setuptools:
+  Latest version by default.
+
+mock:
+  ``1.0.1`` (could be removed in the future).
+
+pillow:
+  ``5.4.1`` (could be removed in the future).
+
+alabaster:
+  ``0.7.x`` (could be removed in the future).
+
+commonmark:
+  ``0.8.1`` (could be removed in the future).
+
+recommonmark:
+  ``0.5.0`` (could be removed in the future).
+
+External dependencies (Conda)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Conda environment:
+
+Conda:
+   Miniconda2 ``4.6.14``
+   (could be updated to use the latest version by default).
+
+Mkdocs:
+  Latest version by default installed via ``conda``.
+
+Sphinx:
+  Latest version by default installed via ``conda``.
+
+sphinx-rtd-theme:
+  Latest version by default installed via ``conda``.
+
+mock:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+pillow:
+  Latest version by default installed via ``pip`` (could be removed in the future).
+
+recommonmark:
+  Latest version by default installed via ``conda`` (could be removed in the future).
+
+Internal dependencies
+~~~~~~~~~~~~~~~~~~~~~
+
+Internal dependencies are needed to integrate your docs with Read the Docs.
+We guarantee that these dependencies will work with all current supported versions of our tools,
+you don't need to specify them in your requirements.
+
+- readthedocs-sphinx-ext
+
+Table of supported versions
+---------------------------
+
+Sphinx
+~~~~~~
+
+Sphinx releases its versions incrementally.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - Released / Latest Update
+     - Supported Until
+
+   * - ``3.x``
+     - Apr 5, 2020 / ???
+     - 5.0 is released or later/early
+
+   * - ``2.x``
+     - Mar 28, 2019 / Mar 5, 2020
+     - 4.0 is released or later/early
+
+   * - ``1.8.x``
+     - Sep 12, 2018 / Mar 10, 2019
+     - Nov 31, 2022
+
+   * - ``1.7.x``
+     - Feb 12, 2018 / Sep 5, 2018
+     - Nov 31, 2022
+
+   * - ``1.6.x``
+     - May 16, 2017 / Feb 4, 2017
+     - Nov 31, 2021
+
+   * - ``1.5.x``
+     - Dec 5, 2016 / May 4, 2017
+     - Nov 31, 2021
+
+   * - ``<= 1.4.x``
+     - Mar 21, 2008 / Nov 23, 2016
+     - Unsupported
+
+Mkdocs
+~~~~~~
+
+MkDocs releases its versions incrementally.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Version
+     - Released / Latest Update
+     - Supported Until
+
+   * - ``1.1.x``
+     - Feb 22, 2020 / ???
+     - 3.0 released or later/early
+
+   * - ``1.0.x``
+     - Aug 3, 2018 / Sep 17, 2018
+     - 2.0 released or later/early
+
+   * - ``0.17.x``
+     - Oct 19, 2017 / Jul 6, 2018
+     - Nov 31, 2021
+
+   * - ``0.16.x``
+     - Nov 4, 2017 / Apr 4, 2017
+     - Nov 31, 2021
+
+   * - ``0.15.x``
+     - Jan 21 2016 / Feb 18, 2016
+     - Nov 31, 2021
+
+   * - ``<= 0.14.x``
+     - Jan 11, 2014 / Jun 9, 2015
+     - Unsupported
+
+Python
+~~~~~~
+
+Python defines an EOL (End Of Life) date for all its versions.
```

Comment:
I think this sentence needs something else, like

```suggestion
Python defines an EOL date for all its versions and we add 6 extra months of support on top of that.
```

However, promise supporting a version of Python after the official EOL date doesn't look good to me. I would prefer if we stick with the official date here: _it could continue working, but we don't compromise ourselves_

Reply:
We only offer 6 months for commercial, 3 months after the EOL in general.